File: Models/Rezervacija.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json;
using System.Text.Json.Serialization;

namespace Models
{
    [Table("Rezervacija")]
    public class Rezervacija
    {
        [Key]
        public int Id {get;set;}
        
        public int KoncertId {get;set;}

        public string Telefon  {get;set;}

        public int Sediste  {get;set;}

    }

}

File: Client/rezervacija.js
export class Rezervacija{
    constructor(id,koncertId,telefon,sediste){
        this.id=id;
        this.koncertId=koncertId;
        this.telefon=telefon;
        this.sediste=sediste;
    }
    async dodajRezervaciju(){
        await fetch(`https:localhost:5001/Rezervacija/DodajRezervaciju/`,{
            method: "POST",
            headers: {"Content-Type": "application/json"},
            body: JSON.stringify({
                koncertId:this.koncertId,
                telefon:this.telefon,
                sediste:this.sediste
            })
        }).then(p => {
            if (p.status == 400) {
                alert("Doslo je do greske.");
            }
        });
    }
};